src/main/charts/confluence/templates/statefulset.yaml

```diff
@@ -71,6 +71,10 @@ spec:
             - name: ATL_TOMCAT_SECURE
               value: "true"
             {{ end }}
+            - name: UMASK
+              value: {{ .Values.confluence.umask | quote }}
```

Comment:
By default, the Confluence docker image creates its log files with file permissions so that only the Confluence user can read them. Change the umask so that they can be read by anyone.

We're assuming here that random punters can't just poke around on the filesystem, so we don't really need restrictive file permissions, but it can always be changed using the values file.